Integrations/PaloAlto_MineMeld/PaloAlto_MineMeld.yml

```diff
@@ -308,24 +308,24 @@ script:
       description: domain
       isArray: false
       name: domain
-      required: true
+      required: false
```

Comment:
change this back to be `true` 
the file validation was fix here https://github.com/demisto/content/pull/4255